src/applets/quicknote/QuickNoteApplet.plugin.in

```diff
@@ -0,0 +1,8 @@
+[Plugin]
+Module=quicknoteapplet.so
+Name=QuickNote
+_Description=Quick & Easy Notes
+Authors=Jacob Vlijm, Budgie Desktop Developers
+Copyright=© 2017-2018 Ubuntu Budgie Developers, 2018 Budgie Desktop Developers
```

Comment:
I don't really feel comfortable with this being copywritten by Ubuntu Budgie. The reasons are:

- I'm not fully aware of Canonical's enforcement policy of their copyright of Ubuntu and thus if this would pose an issue in the future for the project
- "Budgie Desktop Developers" is sufficiently agnostic.